### DISCUSSION TITLE
how to split a video into n segments with fewer overlapping frames while using "Keyframe cut"

### DISCUSSION THREAD

**Discussion by tino926** [mifi/lossless-cut#1485] [Q&A] [answered]
I want to split a video into n parts.

First I use "create num segments" to create segments and press Export
In the Export options view I selected "Keyframe cut"

The problem is that any two adjacent output videos have many overlapping frames.
Is there any option I can choose to avoid these overlapping?

**Comment by mifi** [comment, marked as answer]
can you try to use the "align segments to keyframes" function after making segments?

**Reply by tino926** [reply]
Great, it works! 
The only drawback is that the end point of the last segment has been moved from the end of the full video to previous endpoint. 
This means that several more actions will be needed, but overall, it works.

**Reply by mifi** [reply]
what if you **only** align the start times?

**Reply by tino926** [reply]
This also causes overlapping segments to occur.

**Reply by mifi** [reply]
ah, right 🤦‍♂️
